app-text/poppler/poppler-20.11.0.recipe

```diff
@@ -264,6 +267,8 @@ REQUIRES="
 	lib:libintl$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:liblcms2$secondaryArchSuffix
+	lib:libnspr4$secondaryArchSuffix
+	lib:libnss3$secondaryArchSuffix
```

Comment:
but devel:libnss3$secondaryArchSuffix is commented out

Reply:
It's mentioned in line 385 (BUILD_REQUIRES)